docs/topics/device_connection.md

```diff
@@ -0,0 +1,265 @@
+
+If everything is OK, the system will answer with a 200 OK code, and an empty answer.
+
+This measure will give rise to a Context Entity like the following one in the Context Broker:
+
+      {
+         "type": "Device",
+         "isPattern": "false",
+         "id": "WeatherStation1",
+         "attributes": [
+           {
+             "name": "temperature",
+             "type": "degrees",
+             "value": "22"
+           },
+           {
+             "name": "humidity",
+             "type": "percentage",
+             "value": "78"
+           },
+           {
+             "name": "location",
+             "type": "geo:point",
+             "value": "22.1, -17"
+           },
+           {
+             "name": "reference",
+             "type": "string",
+             "value": "917834508965243"
+           }
+         ]
+      }
+
+### Sending a command
+
+The Platform gives also the opportunity for the devices to receive commands or information from the platform. To illustrate
+this point, we will show how to send some configuration information to a device, using polling commands, i.e.: commands
+that do not require the device to be online to be receieved, but that are retrieved when the device connects to the platform.
+
+Commands are also defined in the device provisioning. It can be seen in the `commands` section of the provisioning payload
+in the provisioning section above.
+
+Information for the command execution is sent using NGSI queries to the Context Broker, as if we were going to update
+directly the Context Entity that represents the device. As an example, we will send the `openUmbrella=false` configuration
+order with the following request:
+
+    PUT /v2/entities
+    Content-Type: application/json
+
+    {
+      "id": "WeatherStation1",
+      "type": "Device",
+      "configuration": {
+        "type" : "command",
+        "value" : "openUmbrella=false"
+      }
+    }
+
+This request will store the command `configuration` with value `openUmbrella=false` waiting for the device to retrieve it.
+
+A device can use the query parameter `getCmd=1` to retrieve all the pending commands from the system whenever it is sending
+a new measure, as in the following example:
+
+    POST /iot/d?k=801230BJKL23Y9090DSFL123HJK09H324HV8732&i=ws1956672&getCmd=1
+    Content-Type: text/plain
+
+    t|27|h|89
+
+Contrary to the previous measurement reports, this one will not be answered with an empty payload, but with a list of all
+the pending commands, as in the following result:
+
+    200 OK
+
+    ws1956672@configuration|openUmbrella=false
+
+Here we can see the configuration value we sent to the Context Broker in the first place. As you can see, the values
+include the Device ID provisioned before, for verification purposes.
+
+Take into account that polling commands (as the ones depicted in this document) have an expiration time, and so, if the
```

Comment:
Default expiration time? Configurable expiration time? Maybe some detail should be provided.